### PR TITLE
fix #38085 filmitorrent.site

### LINKED ISSUE #38085: filmitorrent.site
### Issue URL (Annoyance)
[https://filmitorrent.site/fantastika/3142-dyshi-vo-mgle-2018.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffilmitorrent.site%2Ffantastika%2F3142-dyshi-vo-mgle-2018.html)
### Comment
Username: @Noisia12
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.1.2817.0
Browser: | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:68.0) Gecko/20100101 Firefox/68.0
AdGuard driver: | WFP
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList Dutch,<br>Liste FR,<br>ROList,<br>AdGuard Annoyances,<br>AdGuard French,<br>AdGuard Base,<br>ROLIST2,<br>road-block,<br>NoCoin Filter List,<br>Frellwit's Swedish Filter,<br>Dandelion Sprout's Nordic Filters,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Experimental,<br>AdGuard German,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/38085

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -32,6 +32,7 @@ news.allcrimea.net###kinotvukrnet
 tut.by##div[data-io-title="Новости компаний"]
 masterpaiki.ru###secondary .widget_execphp
 masterpaiki.ru##a[href^="http://epnclick.ru/redirect"]
+filmitorrent.site##a[href^="https://tiruniversal.com"]
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: better block by URL or by more universal selector
`filmitorrent.site##.post > center > a[target="_blank"] > img`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (tvinzz) [reply RussianFilter/sections/specific.txt]: @Alex-302,
changed the rule.

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -32,6 +32,7 @@ news.allcrimea.net###kinotvukrnet
 tut.by##div[data-io-title="Новости компаний"]
 masterpaiki.ru###secondary .widget_execphp
 masterpaiki.ru##a[href^="http://epnclick.ru/redirect"]
+filmitorrent.site##.post > center > a[target="_blank"] > img
```
Comment (AdamWr) [inline RussianFilter/sections/specific.txt]: It seems that this rule doesn't block/hide second ad (please look at the second screenshot here - https://github.com/AdguardTeam/AdguardFilters/issues/38085), so maybe `center > a[target="_blank"] > img` will be okay?

**Review by AdamWr — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (tvinzz) [reply RussianFilter/sections/specific.txt]: @AdamWr,
changed the rule

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**